open-machine-learning-jupyter-book/prerequisites/python-programming-advanced.md

```diff
@@ -188,8 +202,7 @@ assert result == 32
 ### The try statement
 
 "try" statement is used for exception handling.
-When an error occurs, or exception as we call it, Python will normally stop and generate an error
-message. These exceptions can be handled using the try statement.
+When an error occurs, or exception as we call it, Python will normally stop and generate an error message. These exceptions can be handled using the try statement.

```

Comment:
merge or you should add one more empty line between the 204 and 205 If you thin they support to be two lines.